src/hooks/useFetchInitialData.jsx

```diff
@@ -0,0 +1,61 @@
+import { useEffect, useRef } from 'react'
+import { useQuery } from 'cozy-client'
+import {
+  makeAccountsQuery,
+  makeAppsQuery,
+  makeKonnectorsQuery,
+  makeTriggersQuery
+} from 'queries'
+import log from 'cozy-logger'
+
+export const useFetchInitialData = () => {
+  const accountsQuery = useQuery(
+    makeAccountsQuery.definition,
+    makeAccountsQuery.options
+  )
+  const konnectorsQuery = useQuery(
+    makeKonnectorsQuery.definition,
+    makeKonnectorsQuery.options
+  )
+  const appsQuery = useQuery(makeAppsQuery.definition, makeAppsQuery.options)
+  const triggersQuery = useQuery(
+    makeTriggersQuery.definition,
+    makeTriggersQuery.options
+  )
+
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  const allQueries = [accountsQuery, konnectorsQuery, appsQuery, triggersQuery]
+
+  const startTime = useRef(null)
+  const loggedTime = useRef(false)
+
+  useEffect(() => {
+    if (startTime.current === null) {
+      startTime.current = performance.now()
+    }
+
+    const allFetched = allQueries.every(query => query.fetchStatus === 'loaded')
+
+    if (allFetched && !loggedTime.current) {
+      const endTime = performance.now()
+      log('info', `Required data fetched in ${endTime - startTime.current} ms`)
+      loggedTime.current = true
+    }
+  }, [allQueries])
```

Comment:
You want to keep this monitoring in production ?

Reply:
Yes, because it might be useful in the future, and I don't see a strong problem with it, as the impact will be negligible. But I'm open to contradiction if you think this is bad practice

Reply:
It was just to be sure it was not a miss. 

Also, I'm sure there are ways to monitor these kind of things with Sentry. But there's a time for everything.

Reply:
Yes a sailor told me that before he left :)
Let's keep it like this for now, it seems reasonable and might evolve in the future